static/js/pm_list.js

```diff
@@ -62,6 +62,7 @@ function set_pm_conversation_count(user_ids_string, count) {
 
 function remove_expanded_private_messages() {
     stream_popover.hide_topic_popover();
+    ui.destroy_scrollbar($("#private-container"));
     $("#private-container").remove();
```

Comment:
I think we should put this in `if ($("#private-container").length !== 0)`, like the `set_up_scrollbar` call below, rather than adding conditionals into `destroy_scrollbar` itself.

Seems fine otherwise.

Reply:
I'll have to extend the coverage checks. I'm marking this as WIP until i get that fixed. I've squashed the commits.

Reply:
Extended the coverage check.